skt/kernelbuilder.py

```diff
@@ -86,6 +87,24 @@ def prepare(self):
         subprocess.check_call(args)
         self._ready = 1
 
+    def make_redhat_config(self):
+        """Prepare the Red Hat kernel config files."""
+        # These configs build into /redhat/configs
+        args = self.make_argv_base + ['rh-configs']
+        logging.info("building Red Hat configs: %s", args)
+        subprocess.check_call(args)
+
+        # Copy the correct kernel config into place
+        # TODO Add multi-arch support later.
+        config_files = glob.glob(
+            "{}/redhat/configs/kernel*x86_64.config".format(self.source_dir)
```

Comment:
Technically, we need to make sure that `self.source_dir` doesn't have any glob characters. From the `glob` module documentation, the following might work:

    config_files = glob.glob(
        "{}/redhat/configs/kernel*x86_64.config".format(
            re.sub("[]*?[]", "[\g<0>]", self.source_dir)
        )
    )

Please do that or similar escaping.

Reply:
Also, how would `x86_64` play with our multi-arch plan?

Reply:
Support for multi-arch would come later. This does make it a lot easier to select the correct one, though.